src/t_string.c

```diff
@@ -626,24 +626,24 @@ void incrDecrCommand(client *c, long long incr) {
         }
     }
     signalModifiedKey(c,c->db,c->argv[1]);
-    notifyKeyspaceEvent(NOTIFY_STRING,"incrby",c->argv[1],c->db->id);
+    notifyKeyspaceEvent(NOTIFY_STRING, event,c->argv[1],c->db->id);
```

Comment:
The same goes for the following 4 places.
```suggestion
    notifyKeyspaceEvent(NOTIFY_STRING,event,c->argv[1],c->db->id);
```